### PR TITLE
Leaderboard

### PR DESCRIPTION
**Pull request by wes-chen**
Leaderboard functionality. Right now this view is good for 320px screens. Larger screen support will come after design produces those viewports.

- Leaderboard page and its components and corresponding assets
- Updated `Text` object to support font family
- Added selected property for `Button` so there is a "clicked" state
- update website metadata

### CODE REVIEW

**Review by parth4apple — APPROVED** [review]
pls stop using reactstrap

File: src/Leaderboard/LeaderboardList.js [hunk 1]
```diff
@@ -1,34 +1,94 @@
...
+        restroom3_name={props.ranking[2].name}
+        restroom3_src={
+          thumbnails[props.ranking[2]._id]
+            ? thumbnails[props.ranking[2]._id]
+            : "https://i.imgur.com/et7cxmY.png"
+        }
+        restroom3_location={props.ranking[2].location}
+        restroom3_gender={props.ranking[2].gender}
+        restroom3_ratingTotal={props.ranking[2].rating_total}
+        restroom3_numReviews={props.ranking[2].num_ratings}
```
Comment (parth4apple) [inline src/Leaderboard/LeaderboardList.js:67]: Way to many props, pass in as object

File: src/Leaderboard/LeaderboardPage.js [hunk 2]
```diff
@@ -1,48 +1,78 @@
...
-        <LeaderboardTitle />
-        <LeaderboardDescription />
-        <LeaderboardList ranking={this.state.ranking} />
-      </Container>
-    );
-  }
+  return (
+    <div
+      className="text-center"
+      fluid="sm"
```
Comment (parth4apple) [inline src/Leaderboard/LeaderboardPage.js]: Isn't this a reactstrap thing

File: src/Leaderboard/LeaderboardPage.js [hunk 3]
```diff
@@ -1,48 +1,78 @@
...
+      {ranking.length > 0 ? (
+        <LeaderboardList ranking={filteredRanking} total={ranking.length} />
+      ) : (
+        <Loading />
+      )}
+    </div>
+  );
 }
 
-export default withParams(LeaderboardPage);
```
Comment (parth4apple) [inline src/Leaderboard/LeaderboardPage.js:48]: Delete withParams in /utils, don't think we are using it anywhere anymore

File: src/components/Button/style.js [hunk 4]
```diff
@@ -2,6 +2,10 @@ import styled from "styled-components";
 import "@fontsource/montserrat";
 
 export const ButtonStyled = styled.button`
+  display: flex;
+  align-items: center;
+  align-self: center;
+  justify-content: center;
```
Comment (parth4apple) [inline src/components/Button/style.js:8]: Make sure this doesn't break any of the button usages

Comment (parth4apple) [reply src/components/Button/style.js:8]: Although it shouldn't, but double check

File: src/components/LeaderboardCard/style.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+  gap: 4px;
+`;
+
+export const NumberWrapper = styled.div`
+  align-self: center;
+  width: 24px;
+`;
+
+export const LeaderboardCardWrapper = styled.div`
+  width 280px;
```
Comment (parth4apple) [inline src/components/LeaderboardCard/style.js:18]: :

File: src/components/LeaderboardCard/style.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  text-align: left;
+  flex-wrap: nowrap;
+  padding-left: 8px;
+  padding-right: 8px;
+`;
+
+export const LeaderboardCardRatingWrapper = styled.div`
+  display: flex;
+  gap: 8px;
+
```
Comment (parth4apple) [inline src/components/LeaderboardCard/style.js:40]: Nit remove

File: src/components/LeaderboardPodium/index.js [hunk 7]
```diff
@@ -0,0 +1,273 @@
...
+  restroom2_numReviews,
+  restroom2_relativeRank,
+  restroom3_id,
+  restroom3_name,
+  restroom3_src,
+  restroom3_location,
+  restroom3_gender,
+  restroom3_ratingTotal,
+  restroom3_numReviews,
+  restroom3_relativeRank
```
Comment (parth4apple) [inline src/components/LeaderboardPodium/index.js:46]: Too many props -- can be left to fix later

File: src/components/LeaderboardPodium/style.js [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  justify-content: center;
+  align-items: center;
+  gap: 16px;
+`;
+
+export const PodiumCard = styled.div`
+  grid-column-start: 1;
+  grid-column-end: 2;
+  grid-row-start: 1;
+  grid-row-end: 2;
```
Comment (parth4apple) [inline src/components/LeaderboardPodium/style.js:33]: Do you need this, I don't see display grid

File: src/Leaderboard/LeaderboardList.js (hunk 1 above)
Comment (wes-chen) [reply src/Leaderboard/LeaderboardList.js:67]: leaving as TODO

File: src/Leaderboard/LeaderboardPage.js (hunk 2 above)
Comment (wes-chen) [reply src/Leaderboard/LeaderboardPage.js]: yes byebye

File: src/Leaderboard/LeaderboardPage.js (hunk 3 above)
Comment (wes-chen) [reply src/Leaderboard/LeaderboardPage.js:48]: yes byebye

File: src/components/LeaderboardCard/style.js (hunk 5 above)
Comment (wes-chen) [reply src/components/LeaderboardCard/style.js:18]: ?

File: src/components/LeaderboardCard/style.js (hunk 6 above)
Comment (wes-chen) [reply src/components/LeaderboardCard/style.js:40]: done

**Pull request merged**